schedule/functional/upgrade_Leap_15.1_gnome.yaml

```diff
@@ -0,0 +1,81 @@
+---
+name: upgrade_Leap_15.1_gnome
+description: >
+    Maintainer: zluo
+    Upgrade test Leap 15.1 to TW  and rollback after upgrade
+schedule:
+    - installation/isosize
+    - installation/bootloader
+    - installation/welcome
+    - installation/upgrade_select
+    - installation/online_repos
+    - installation/resolve_dependency_issues
+    - installation/installation_overview
+    - installation/disable_grub_timeout
+    - installation/start_install
+    - installation/await_install
+    - installation/logs_from_installation_system
+    - installation/reboot_after_installation
+    - installation/grub_test
+    - installation/first_boot
+    - installation/opensuse_welcome
+    - console/system_prepare
+    - update/zypper_clear_repos
+    - console/zypper_ar
+    - console/zypper_ref
+    - console/check_network
+    - console/system_state
+    - console/prepare_test_data
+    - console/consoletest_setup
+    - locale/keymap_or_locale
+    - console/force_scheduled_tasks
+    - console/textinfo
+    - console/hostname
+    - console/x_vt
+    - console/zypper_lr
+    - console/ncurses
+    - console/yast2_lan
+    - console/curl_https
+    - console/salt
+    - console/glibc_sanity
+    - console/zypper_in
+    - console/zypper_log
+    - console/yast2_i
+    - console/yast2_bootloader
+    - console/vim
+    - console/firewall_enabled
+    - console/sshd
+    - console/ssh_cleanup
+    - console/sshfs
+    - console/mtab
+    - console/orphaned_packages_check
+    - console/consoletest_finish
+    - x11/user_gui_login
+    - x11/desktop_runner
+    - x11/xterm
+    - locale/keymap_or_locale_x11
+    - x11/sshxterm
+    - x11/gnome_control_center
+    - x11/gnome_tweak_tool
+    - x11/gnome_terminal
+    - x11/gedit
+    - x11/firefox
+    - x11/firefox_audio
+    - x11/chromium
+    - x11/graphicsMagick
+    - x11/ooffice
+    - x11/oomath
+    - x11/oocalc
+    - x11/glxgears
+    - x11/nautilus
+    - x11/gnome_music
+    - x11/evolution
+    - x11/desktop_mainmenu
+    - x11/inkscape
+    - x11/gimp
+    - x11/hexchat
+    - x11/vlc
+    - x11/reboot_gnome
+    - installation/boot_into_snapshot
+    - installation/rollback_from_ro_snapshot
+    - shutdown/shutdown
```

Comment:
@Zaoliang, can you alter the schedule, and add:

```
    - console/system_prepare
    - console/check_network
    - console/system_state
    - console/prepare_test_data
    - console/consoletest_setup
    - locale/keymap_or_locale
    - console/force_scheduled_tasks
     - x11/firefox 
     - x11/nautilus 
     - x11/reboot_gnome 
     - shutdown/shutdown 
```

I'm wondering if it's a change within the SUT that causes the problem with the test and alt-f2 switching to a console instead

Reply:
I thought you discarded this hypothesis via manual test.

Reply:
Via manual test, I could confirm that there's no product bug :), but the vnc connection is quite bad. Let's use openQA to verify what's going on in there.

Reply:
tried your suggestion:

http://10.162.23.47/tests/8416#step/reboot_gnome#1/13

failed already on reboot_gnome, this might be performance issue? I tested this a couple of times, thanks!

Reply:
@Zaoliang add again the changes to the schedule that I suggested, and in [tests/x11/firefox.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/x11/firefox.pm#L26), add the following change:

`select_console 'x11'; `

This is what's making the last run to fail

Reply:
tried this as well but it failed:

http://10.162.23.47/tests/8437#step/firefox#1/3